resources/wikia/modules/iframeWriter.js

```diff
@@ -17,6 +17,7 @@ define('wikia.iframeWriter', [
 	 * @param {string} params.code - code to put into the iframe
 	 * @param {number} params.width - desired width of the iframe
 	 * @param {number} params.height - desired height of the iframe
+	 * @param {string} [params.classes] - custom class(es) of the iframe (coma-separated if more than one)
```

Comment:
1. It's a notation we use in AdEng - square brackets = optional parameter and PhpStorm accepts it :smile_cat: 
2. Good catch! I'll correct it in a minute :cake: